### PR TITLE
Update unmaintained status to py-data and minor fix to script

### PR DESCRIPTION
**Pull request by zhihao11ui**

### CODE REVIEW

File: py-data.csv
Comment (darko-marinov) [inline py-data.csv:1]: It looks as if you changed each and every line here. Was there some issue with the EOL?

Comment (zhihao11ui) [reply py-data.csv:1]: Thanks, I fixed the issue and that is exactly why

**Comment by darko-marinov** [comment]
Change ` | 2020/01/17` to `;2020/01/17` (although even better would be `latest commit 2020/01/17`). Did you have any case where a test already has some non-blank status (e.g., `Opened`) but you wanted to add `Unmaintained`?

**Comment by zhihao11ui** [comment]
For any non-blank status, I just skip them

**Comment by darko-marinov** [comment]
Fine to skip non-blank. Please change those vertical bars to semicolons in all rows.

**Comment by darko-marinov** [comment]
For `mojeek` you removed double quotes. Please return them back (or even better open first one PR to remove those double quotes and the TAB character that shouldn't have been there anyhow).

**Comment by darko-marinov** [comment]
I see people working on unmaintained repos, so let me accept your PR even with its small mistakes. We can clean it later.

**Pull request merged**